go/enclave/rpc_server.go

```diff
@@ -37,6 +37,11 @@ func StartServer(
 	erc20ContractLib erc20contractlib.ERC20ContractLib,
 	collector StatsCollector,
 ) (func(), error) {
+	tomlConfig, err := toml.Marshal(enclaveConfig)
```

Comment:
Unrelated but v. minor - it felt right to detect immediately if printing the config would fail, rather than after starting a server.